testcases/OpTestEEH.py

```diff
@@ -300,18 +311,24 @@ def runTest(self):
                 print "=================Injecting the fenced PHB error on PHB: %s=================" % domain
                 l_con.run_command(cmd)
                 # Give some time to EEH PCI Error recovery
-                time.sleep(30)
-                self.gather_logs()
-                if i == 0:
-                    if not self.check_eeh_phb_recovery(domain):
-                        raise EEHRecoveryFailed("PHB domain", domain)
-                    else:
-                        print "PHB %s recovery successful" % domain
-                else:
-                    if self.check_eeh_phb_recovery(domain):
-                        raise EEHRecoveryFailed("PHB domain", domain)
+                tries = 30
+                recovery_done = False
+                for i in range(1,tries):
+                    time.sleep(1)
+                    if i == 0:
```

Comment:
Can you change i variable to j here in inner for loop, as it conflicts with outer for loop i variable which we are using it for checking for max EEH(2 times)